services-camel/src/test/java/edu/unc/lib/dl/services/camel/solrUpdate/SolrUpdateProcessorIT.java

```diff
@@ -117,7 +117,7 @@ public void testReindexAcls() throws Exception {
 
         processor.process(exchange);
 
-        assertTrue(notify.matches(3l, TimeUnit.SECONDS));
+        assertTrue(notify.matches(6l, TimeUnit.SECONDS));
```

Comment:
Why has the time increased?

Reply:
It was taking longer to complete in travis